src/styles/index.scss

```diff
@@ -2,2 +1,86 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-color: #00acdc;
+}
+
+.body {
```

Comment:
The class `.body` might be confused with the HTML `body` tag. It's better to use a different class name to avoid confusion and potential styling issues.